buildSrc/build.gradle.kts

```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 dependencies {
-    implementation(kotlin("gradle-plugin-api", version = "1.4.21"))
+    implementation(kotlin("gradle-plugin-api", version = "1.5.0"))
```

Comment:
U sure this works? buildSrc is really slow at upgrading the Kotlin version and if it's to new it get's mad

Reply:
[The version seems to exist](https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-gradle-plugin-api), the CI doesn't fail because of it and building it locally with `gradle clean build --no-build-cache` doesn't seem to produce any errors. 

If you have any issues building this yourself because of this, please do share a stacktrace and I'll look into it.

Reply:
that's not the problem the version exist as soon as the new gradle plugin version is out
We need to wait for the gradle release adding support for the Kotlin version but if the build works for you i guess it works now